content/en/docs/concepts/cluster-administration/priority-and-fairness.md

```diff
@@ -0,0 +1,123 @@
+This includes not only kube-apiservers but aggregated apiservers too.
+Prioritization and fairness applies only to requests that are not
+_long running_.  Long running requests are WATCHes and EXECUTEs.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Status
+
+This feature is named "RequestManagement" and is alpha in
+release 1.16.  As an alpha feature, it is disabled by default.  When
+enabled it replaces a request filter that enforces simple aggregate
+concurrency limits with this feature, which can be seen as a more
+general version of that earlier functionality.
+
+## Outline
+
+Each apiserver independently applies prioritization and fariness,
+based on API objects that configure this feature.  This involves two
+concepts: priority level and flow schema.
+
+### Priority level
+
+Each apiserver has an aggregate concurrency limit and divides it ---
+without examining the request load --- among several priority levels.
+The priority levels operate independently of each other.  That is, the
+apiserver classifies a given request to exactly one priority level
+and, for a given priority level, handles at most the corresponding
+number of requests at any given time.  One special priority level ---
+used for meta-requests and other top-priority traffic --- imposes no
+limit.  This priority level and the requests mapped to it are
+described as "exempt", because of the lack of concurrency limitation.
+
+For a non-exempt request, if it is not allowed to be handled
+immediately then the apiserver will either reject the request or hold
+it in a queue for a while.  Each non-exempt priority level has a
+number of FIFO queues.  Whenever that priority level's number of
+requests currently being handled drops below the concurrency limit,
+the apiserver draws requests to begin handling from these queues using
+a technique that is an appropriate generalization of
+[fair queuing](https://en.wikipedia.org/wiki/Fair_queuing).
+
+### Flow schema
+
+To determine how to control a request, an apiserver first classifies
+that request to a _flow_ according to a configured collection of _flow
+schemas_.  The flow schemas are considered in order, according to each
+flow schema's _matching precedence_ (which is a number).  A flow
+schema has rules that describe which requests match that schema.
+Matching takes into account (a) the requested operation --- as
+identified by the HTTP request verb and URI --- and (b) if the request
+arrived through a secured port then the authenticated user attributes
+otherwise just the UserAgent.  Among the schemas that match the
+request, one with the logically highest matching precedence wins.
+
+A flow schema also has a declaration of how to complete the _flow
+identifier_ for the request's flow.  A flow identifier is a pair of
+strings: the name of its flow schema plus one more string called the
+_flow distinguisher_.  A flow schema indicates whether the
+distinguisher is (1) the username or UserAgent of the user making the
```

Comment:
If something other than a human user makes a request (eg a Pod), does that client's identity contribute to the _flow distinguisher_?

Reply:
I thought the language here answers that question.  It speaks of requests without limitation to those made by human using kubectl.  The sentence commented upon says that what contributes to the flow distinguisher is declared by the flow schema.

Reply:
These docs usually use the term “user” to mean a human, so I'm keen to make sure the language helps a reader who (eg) hasn't yet come across ServiceAccount as a concept.